From 5d338c5d37335a7e6f521448fdc3552943466930 Mon Sep 17 00:00:00 2001
From: Alistair Strachan <alistair.strachan@imgtec.com>
Date: Mon, 10 Mar 2014 17:00:25 -0700
Subject: [PATCH 343/634] video: adf: export the adf_attachment_allow symbol to
 modules.

There are no in-tree users of adf_attachment_allow, but out-of-tree
modules want to use it. It looks like this function should be
EXPORT_SYMBOL.

Change-Id: Iad522dc5d32ac09fec6483bbc317db8ecae12e97
Signed-off-by: Alistair Strachan <alistair.strachan@imgtec.com>
---
 drivers/video/adf/adf.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/adf/adf.c b/drivers/video/adf/adf.c
index 933e74a..2d20024 100644
--- a/drivers/video/adf/adf.c
+++ b/drivers/video/adf/adf.c
@@ -920,6 +920,7 @@ done:
 
 	return ret;
 }
+EXPORT_SYMBOL(adf_attachment_allow);
 
 /**
  * adf_obj_type_str - string representation of an adf_obj_type
-- 
1.9.1

